defaults/main.yml

```diff
@@ -3,4 +3,6 @@
 ufw_rules_to_create: []
 ufw_rules_to_delete: []
 
-ufw_default_policy: deny
+ufw_default_policy:
+  - { direction: "incoming", policy: "deny" }
+  - { direction: "outgoing", policy: "deny" }
```

Comment:
I don't think this means what you want it to mean. Outgoing connections are not blocked by default in the current setup.

Reply:
Ahh! I removed the outgoing rule. :)